generators/client/templates/angular/src/main/webapp/app/layouts/profiles/profile.service.ts.ejs

```diff
@@ -32,7 +32,7 @@ export class ProfileService {
     constructor(private http: HttpClient) { }
 
     getProfileInfo(): Promise<ProfileInfo> {
-        if (!this.profileInfo) {
+        if (this.profileInfo === undefined) {
```

Comment:
if it's always undefined, okay. what about null? no chance of happening?

Reply:
yes, I expect it to be always `undefined` as that's default Javascript variable value.